### PR TITLE
PIM-10215: add translations

### PR DESCRIPTION
**Pull request by guillaumefer-akeneo**
**Description (for Contributor and Core Developer)**
https://akeneo.atlassian.net/browse/PIM-10215

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by anaisbaunelemaire — APPROVED** [review]

**Comment by anaisbaunelemaire** [comment]
Don't forget the changelog ;-)

**Review by ghoussard — APPROVED** [review]

File: src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml [hunk 1]
```diff
@@ -174,6 +174,8 @@ pim_import_export:
                 compute_completeness_of_products_family: Compute completeness of products family
                 mass_delete: Mass delete products
                 clean_removed_attribute_job: Cleaning removed attribute values
+                asset_manager_mass_delete_assets: Mass Delete
+                reference_entity_mass_delete_records: Mass Delete
```
Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml:178]: Could you add asset_manager_mass_edit_assets please ?

Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml:178]:
```suggestion
                asset_manager_mass_delete_assets: Mass delete assets
                reference_entity_mass_delete_records: Mass delete records
```

File: src/Akeneo/Platform/Job/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml [hunk 2]
```diff
@@ -40,6 +40,8 @@ akeneo_job_process_tracker:
     compute_completeness_of_products_family: Compute completeness of products family
     clean_removed_attribute_job: Cleaning removed attribute values
     execute_rules: Execute rules
+    asset_manager_mass_delete_assets: Mass Delete
+    reference_entity_mass_delete_records: Mass Delete
```
Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Job/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml:44]:
```suggestion
    asset_manager_mass_delete_assets: Mass delete assets
    reference_entity_mass_delete_records: Mass delete records
```

**Review by StevenVAIDIE — COMMENTED** [review]
Do not forget to modify CHANGELOG.md

File: src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml (hunk 1 above)
Comment (guillaumefer-akeneo) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml:178]: This is a request from the ticket (Author + PM). On the front we already have the information about the source : 
image

File: src/Akeneo/Platform/Job/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml (hunk 2 above)
Comment (guillaumefer-akeneo) [reply src/Akeneo/Platform/Job/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml:44]: Same here

File: src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml (hunk 1 above)
Comment (guillaumefer-akeneo) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml:178]: It's added :)
image
image

Comment (StevenVAIDIE) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml]: If you keep like this you will have in the type filter three time "Mass Delete" and you never know witch one it is

Comment (guillaumefer-akeneo) [reply src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml]: :thinking: I'm asking the question

**Comment by RiffVentura** [comment]
It's a bit odd to have EE features mention in CE.
Shouldn't it be [here](https://github.com/akeneo/pim-enterprise-dev/blob/ac0f1c1c061406755af871873afc39983c2638eb/src/Akeneo/AssetManager/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml#L582) instead ? or [here](https://github.com/akeneo/pim-enterprise-dev/blob/ac0f1c1c061406755af871873afc39983c2638eb/src/Akeneo/ReferenceEntity/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml#L373) ?
cc @StevenVAIDIE

**Comment by ghoussard** [comment]
> It's a bit odd to have EE features mentioned in CE. Shouldn't it be [here](https://github.com/akeneo/pim-enterprise-dev/blob/ac0f1c1c061406755af871873afc39983c2638eb/src/Akeneo/AssetManager/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml#L582) instead ? or [here](https://github.com/akeneo/pim-enterprise-dev/blob/ac0f1c1c061406755af871873afc39983c2638eb/src/Akeneo/ReferenceEntity/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml#L373) ? cc @StevenVAIDIE

Agree with you

**Comment by RiffVentura** [comment]
Moving onto EE repository
https://github.com/akeneo/pim-enterprise-dev/pull/12620

**Pull request closed without merging**